src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -603,10 +606,467 @@ def paramsToList(params, config) {
 }
 
 def paramsToChannel(params, config) {
+  System.err.println("Warning: paramsToChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   Channel.fromList(paramsToList(params, config))
 }
 
 def viashChannel(params, config) {
+  System.err.println("Warning: viashChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   paramsToChannel(params, config)
```

Comment:
VDSL3 components currently use `viashChannel()`, so it would be weird that the output of a `viash build -p nextflow` would generate deprecation warnings when we're the ones who created it ;) Can you change the VDSL3 helper so that it uses `channelFromParams()` and `preprocessInputs` instead?

Reply:
Done, added to the VDSL3 helper